src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/model/policy/AdditionalConditionAp.kt

```diff
@@ -4,10 +4,10 @@ data class AdditionalConditionAp(
   override var code: String,
   override val category: String,
   override val text: String,
-  override val tpl: String?,
```

Comment:
this just means that you don't have to specify `blah = null` when creating new instances

Reply:
which massively shortens the policy files below